proteus/MeshAdaptPUMI/MeshConverter.cpp

```diff
@@ -941,6 +941,18 @@ int MeshAdaptPUMIDrvr::reconstructFromProteus(Mesh& mesh, Mesh& globalMesh,int h
   numModelTotals[2] = numModelBoundaries;
   numModelTotals[3] = 0;
 
+  //Get Region starting material
+  entIter = m->begin(numDim);
+  int regStartMaterial = 100;
+  int rID = 0;
+  while(ent = m->iterate(entIter)){
+    if(mesh.elementMaterialTypes[rID] < regStartMaterial)
+      regStartMaterial = mesh.elementMaterialTypes[rID];
+    rID++;
+  }
+  m->end(entIter);
+
+
```

Comment:
Yes, now it shouldn't matter what you set as the region flag in the domain construction. Although oddly, it's only with the floating bar case that I've seen that the region flag set is not equivalent to the elementMaterialType.